.github/workflows/build.yml

```diff
@@ -15,40 +18,52 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: cisagov/setup-env-github-action@develop
-      - uses: actions/checkout@v1
-      - uses: actions/setup-python@v1
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
         with:
-          python-version: 3.7
-      - name: Cache pip test requirements
-        uses: actions/cache@v1
+          python-version: 3.8
+      - uses: actions/setup-go@v2
         with:
-          path: ${{ env.PIP_CACHE_DIR }}
-          key: "${{ runner.os }}-pip-test-\
-            ${{ hashFiles('**/requirements-test.txt') }}"
-          restore-keys: |
-            ${{ runner.os }}-pip-test-
-            ${{ runner.os }}-pip-
-      - name: Cache pre-commit hooks
-        uses: actions/cache@v1
-        with:
-          path: ${{ env.PRE_COMMIT_CACHE_DIR }}
-          key: "${{ runner.os }}-pre-commit-\
-            ${{ hashFiles('**/.pre-commit-config.yaml') }}"
-      - name: Cache curl downloads
-        uses: actions/cache@v1
-        with:
-          path: ${{ env.CURL_CACHE_DIR }}
-          key: "${{ runner.os }}-curl-\
-            terraform-${{ env.TERRAFORM_VERSION }}"
+          go-version: '1.14.2'
+      - name: Store installed Python version
+        run: |
+          echo "::set-env name=PY_VERSION::"\
+          "$(python -c "import platform;print(platform.python_version())")"
+      - name: Store installed Go version
+        run: |
+          echo "::set-env name=GO_VERSION::"\
+          "$(go version | sed 's/^go version go\([0-9.]\+\) .*/\1/')"
```

Comment:
`set-env` changes needed.  Now or later...

Reply:
It should pull them down when https://github.com/cisagov/skeleton-tf-module/pull/35 is merged.

Reply:
I'd rather let Linage generate a PR to merge them automatically vs possibly introducing issues with `git` history.

Reply:
`git` will sort it out, but I'm fine either way.